pkg/trace/api/info_test.go

```diff
@@ -163,8 +155,8 @@ func TestInfoHandler(t *testing.T) {
 		}
 	}
 }` {
-		t.Fatal("Output of /info has changed. Changing the keys " +
-			"is not allowed because the client rely on them and " +
-			"is considered a breaking change")
+		t.Fatal("Output of /info has changed. Changing the keys "+
```

Comment:
I think that only the endpoint list + version + abilityToDropP0 should be enforced. These are the only  informations used by the tracer.
- drop configs / feature section from the endpoint (I prefer this option)
- nest both in a "debug" dict {endpoints:[], debug:{configs:.., features:..}}. This keeps debugging info and separates clearly specifics of the agent from tracer<->agent dependencies
The hash can be computed without agent specific fields and contain only informations relevant to the tracer

Reply:
I thought we already agreed with the team that this is all that we want. @dougqh also expressed that he'd like to know details about the agent including maximum accepted payload size etc. I would not make any more changes to the output. I think this is all useful information which we may end up wanting later on. Better have it from the start, a few bytes don't hurt anyone.